projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -291,6 +291,7 @@ private function build_task( $task ) {
 		$built_task['subtitle']     = $this->load_subtitle( $task );
 		$built_task['badge_text']   = $this->load_value_from_callback( $task, 'badge_text_callback' );
 		$built_task['isLaunchTask'] = isset( $task['isLaunchTask'] ) ? $task['isLaunchTask'] : false;
+		$built_task['extra_data']   = $this->load_extra_data( $task );
```

Comment:
Rather than assigning this in all cases, can we only return it when we have extra data? That may be easier if the task returns an array or null, but it means we only return the data for `extra_data` over the wire when we have something useful to include. Also, `array()` serialised to JSON doesn't always play nicely with client expectations for JSON objects vs arrays.
```suggestion
		$extra_data                 = $this->load_extra_data( $task );
		if ( is_array( $extra_data ) ) {
			$built_task['extra_data'] = $extra_data;
		}
```